.github/workflows/metadata-ingestion.yml

```diff
@@ -54,14 +54,20 @@ jobs:
       DATAHUB_LOOKML_GIT_TEST_SSH_KEY: ${{ secrets.DATAHUB_LOOKML_GIT_TEST_SSH_KEY }}
     strategy:
       matrix:
-        python-version: ["3.7", "3.10"]
         command:
           [
             "installAirflow1",
             "testIntegration",
             "testIntegrationBatch1",
             "testSlowIntegration",
           ]
+        include:
+          - {
+              python-version: "3.7",
+              extraPythonRequirement: "sqlalchemy==1.3.24",
```

Comment:
@hsheth2 you need to use `>=` instead of `==` to match the rest of the changes

Reply:
This is in the CI test matrix, where we want to test that we remain compatible with SQLAlchemy 1.3.24.